mappings/net/minecraft/world/gen/surfacebuilder/StoneShoreSurfaceBuilder.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_6485 net/minecraft/world/gen/surfacebuilder/StoneShoreSurfaceBuilder
```

Comment:
This is also used by stony peaks (as parent) iirc - I mapped it `StoneSurfaceBuilder`.